### DISCUSSION TITLE
Why cant we acess animation clip entitypaths?

### DISCUSSION THREAD

**Discussion by Sirmadeira** [bevyengine/bevy#13522] [Q&A]
[Animation clip docs ](https://docs.rs/bevy/latest/bevy/animation/prelude/struct.AnimationClip.html)

In the docs we can easily access the curves that gives us the interpolation of keyframes and so on, which btw super usefull thank to whoever did that really helped me with my rag doll physics.

But we cant access the entity paths itself anyone know the reason? Could be really neat to adjust the paths to another entity and so on

**Comment by m11m** [comment]
Did you find a way to do this? I would also find it useful.

**Reply by Sirmadeira** [reply]
Bevy has refactored the bevy animation crate, they created a new component AnimationTargetId and got rid of entity paths in general the new version of structs is way better. If your project doesnt have lots of dependencies, you can try using main directly.

**Reply by m11m** [reply]
Awesome, thanks
